app/kafka/manager/utils/AdminUtils.scala

```diff
@@ -238,6 +237,18 @@ class AdminUtils(version: KafkaVersion) {
     createOrUpdateTopicPartitionAssignmentPathInZK(curator, topic, newPartitionsReplicaList, update=true, readVersion=readVersion)
   }
 
+  /* Add partitions to multiple topics. After this operation, all topics will have the same number of partitions */
+  def addPartitionsToTopics(curator: CuratorFramework,
+                            topicAndReplicaList: Seq[(String, Map[Int, Seq[Int]])],
+                            newNumPartitions: Int,
+                            brokerList: Seq[Int],
+                            readVersions: Map[String,Int]) {
+    // topicAndReplicaList is sorted by number of partitions each topic has in order not to start adding partitions if any of requests doesn't work with newNumPartitions
+    for ((topic, replicaList) <- topicAndReplicaList) {
+      addPartitions(curator, topic, newNumPartitions, replicaList, brokerList, readVersions.getOrElse(topic,-1))
```

Comment:
Probably should skip topics for which we don't have a readVersion to avoid potential issues with two concurrent add partitions to topics commands.  You can return a list of topics for which no read version was found or some form of error object.